Modules/IO/GDCM/include/itkGDCMImageIO.h

```diff
@@ -164,6 +164,12 @@ class ITKIOGDCM_EXPORT GDCMImageIO : public ImageIOBase
   itkGetConstMacro(LoadPrivateTags, bool);
   itkBooleanMacro(LoadPrivateTags);
 
+  /** Convert YBR to RGB
+   */
+  itkSetMacro(YBRtoRGB, bool);
+  itkGetConstMacro(YBRtoRGB, bool);
+  itkBooleanMacro(YBRtoRGB);
```

Comment:
I don't think this option/IVAR is needed. The expectation is that if the ImageIO's PixelType is RGB, then the image buff is RGB not YBR.

If this IVAR is kept than it will need to be added to the "PrintSelf" method.

Reply:
Please kindly preserve the option, it was already discussed in the very first PR. Thank you. There is some advantage to take only Y or something like and it will be backward compatible if someone have already some workaround, IMHO.

Reply:
Yes it was discussing in the other PR: https://github.com/InsightSoftwareConsortium/ITK/pull/822

My conclusion is that the ImageIO::PixelType must be respected and consistent with RGB. Hence the current ITK behavior is a bug, and requires correction. If one wants a YBR buffer that must not be reported as an RGB PixelType by ImageIO. 

The recommendation is to remove the option, otherwise there will need to be more effort in ensure a YBR buffer is consistent with the ImageIO state.

Reply:
> We should add a test or two for these cases so that we know if the behavior in GDCM changes with versions.

BTW, that option will allow very easy tests what is true and what is not, and workaround even behavior for lossy Jpeg will change in GDCM

Reply:
S. [post](https://github.com/InsightSoftwareConsortium/ITK/pull/822#issuecomment-490552962) 
 in old PR

Reply:
E.g. example for that option, access as vector[3] image and extract luma.

```
#include <itkImage.h>
#include <itkVectorImage.h>
#include <itkImageFileReader.h>
#include <itkImageFileWriter.h>
#include <itkGDCMImageIO.h>
#include <itkVectorIndexSelectionCastImageFilter.h>
// Input Y'CbCr color-space image, output is Y' scalar image (luma)
int main(int argc, char ** argv)
{
  typedef itk::VectorImage<unsigned char, 3> VectorImageType;
  typedef itk::Image<unsigned char, 3> ImageType;
  typedef itk::ImageFileReader<VectorImageType> ReaderType;
  typedef itk::ImageFileWriter<ImageType> WriterType;
  itk::GDCMImageIO::Pointer imageIO = itk::GDCMImageIO::New();
  ReaderType::Pointer reader = ReaderType::New();
  //imageIO->SetInputYBRtoRGB(false); <----------------------------------------------------
  reader->SetImageIO(imageIO);
  reader->SetFileName(argv[1]);
  reader->Update();
  typedef itk::VectorIndexSelectionCastImageFilter<VectorImageType, ImageType> FilterType;
  FilterType::Pointer filter = FilterType::New();
  filter->SetIndex(0);
  filter->SetInput(reader->GetOutput());
  filter->Update();
  WriterType::Pointer writer = WriterType::New();
  writer->SetUseCompression(true);
  writer->SetFileName("luma.tiff");
  writer->SetInput(filter->GetOutput());
  writer->Update();
  return 0;
}
```